applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -330,9 +460,8 @@ static void cmd_send(struct k_work *work)
 	}
 
 done:
-	k_sleep(K_MSEC(100)); /* allow time for TX DMA */
-	buf_num = 1U;
-	err = uart_rx_enable(uart_dev, &uart_rx_buf[0], 1, SYS_FOREVER_MS);
+	err = uart_rx_enable(uart_dev, uart_rx_buf[0],
+				sizeof(uart_rx_buf[0]), 1);
```

Comment:
Please add a define for the UART RX stop timeout (1ms)

Reply:
Define UART_RX_TIMEOUT to represent timeout.
#define UART_RX_TIMEOUT 1